cmd/common.go

```diff
@@ -75,6 +76,24 @@ func AddCommonFlags(cmd *cobra.Command, cfg *CommonConfig) {
 	}
 }
 
+var (
+	// clusterIdentifier provides a Prometheus metric that should uniquely identify the
+	// cluster that any stolon component is associated with. Users can then join between
+	// various metric series for the same cluster without making assumptions about service
+	// discovery labels.
+	clusterIdentifier = prometheus.NewGaugeVec(
```

Comment:
This is the metric we (GoCardless) have been using to join various time series together. By having all components report this in a consistent fashion (see [here](https://github.com/gocardless/stolon-pgbouncer/blob/master/cmd/stolon-pgbouncer/main.go#L132-L138) for the stolon-pgbouncer definition) it becomes possible to join series from totally different process/infrastructures on the cluster_name and store_prefix labels.

Reply:
Ideally only the clustername should be neeed (the prefix isn't used when running stolon inside k8s). Obviously people could setup multiple instances using the same cluster name and prefix using different store instances. But I think people that use multiple stolon instances should also use different cluster names if they want to distinguish them inside prometheus.

Another solution will be to add another "clusteruid" option but I think this is redundanto and should be covered by the "clustername"